pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -115,47 +114,6 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) (opErr error) {
 	return nil
 }
 
-// GetIP returns the control plane endpoint for the cluster.
-func (a *Actuator) GetIP(cluster *clusterv1.Cluster, machine *clusterv1.Machine) (string, error) {
-	clusterContext, err := context.NewClusterContext(&context.ClusterContextParams{
-		Cluster:    cluster,
-		Client:     a.client,
```

Comment:
Loosing access to clients here, guessing that's okay in this case?

Reply:
Yes. I realized that the deployer was never supposed to have that access in the first place. It was just the way CAPA was built. The deployer simply builds a kubeconfig or parses the control plane endpoint out of one of those objects.